tls/s2n_tls12_encrypted_extensions.c

```diff
@@ -16,6 +16,7 @@
 #include "error/s2n_errno.h"
 #include "utils/s2n_safety.h"
 #include "stuffer/s2n_stuffer.h"
+#include "tls/extensions/s2n_npn.h"
 
```

Comment:
Why do you need this include?

Reply:
Because I'm calling an extension function now. Otherwise how would this file know what s2n_npn_encrypted_extension.recv() is?

Reply:
Oh right! For some reason I was reading that as pulling in the new header 🤦‍♀️